aether_server/routes/views/ws_view.py

```diff
@@ -1,15 +1,17 @@
-import json
-
 import aiohttp.web as web
 
+import json
+
 from aether_server.routes.routes_decl import generic_routes
+from aether_server.routes.views.rtc_view import AetherRtc
 
 
 @generic_routes.view("/ws")
 class AetherWSView(web.View):
     async def get(self):
         ws = web.WebSocketResponse()
         await ws.prepare(self.request)
+        rtc = AetherRtc()
```

Comment:
TODO: Change directories for the RTC class instead of `view`. Possibly within a `.utils` sub-module.